next/components/forms/segments/AccountSections/TaxesFeesSection/TaxDetails.tsx

```diff
@@ -73,8 +73,8 @@ const TaxDetails = () => {
           </div>
         </div>
         <div className="h-0.5 w-full bg-gray-800" />
-        <div className="flex w-full flex-col gap-2 xs:flex-row lg:gap-6">
-          <div className="text-h4 w-full grow xs:w-min">{t('tax_detail_section.tax_to_pay')}</div>
+        <div className="xs:flex-row flex w-full flex-col gap-2 lg:gap-6">
```

Comment:
Again suspicious order. Is `xs` even defined?

Reply:
I realized I've provided wrong path to Tailwind theme CSS.
03c39214112cdde179b8312b2f096392cc2f71c3